setup-gcloud-sdk/action.yml

```diff
@@ -0,0 +1,28 @@
+name: "gcloud sdk setup"
+author: "@andreavs"
+description: "An action to setup gcloud specific to hagal"
+branding:
+  icon: "truck"
+  color: "gray-dark"
+inputs:
+  gcp_credentials:
+    description: "Google cloud service account key"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - id: 'auth'
+      uses: 'google-github-actions/auth@v0'
+      with:
+        credentials_json: '${{ inputs.gcp_credentials }}'
+
+    - name: 'Set up Cloud SDK'
+      uses: 'google-github-actions/setup-gcloud@v0'
+
+    - run: |
+        gcloud config set artifacts/repository hagal
+        gcloud config set artifacts/location europe-west1
+        gcloud auth configure-docker europe-docker.pkg.dev
```

Comment:
Same here, not sure how our artifacts locatio and repo data is sensitive.  Leaving it to your judgement.

Reply:
I think i will leave this as is - it does not reveal too much, only our name and that we are located in europe.